EphemeralElixir/EphemeralElixir
# Add disconnect handler for socket io

#### What does this PR do?

Adds functionality for when a user disconnects -- they will be removed from activeUsers object
#### Where should the reviewer start?

In socket.server.js
#### How should this be manually tested?

Use multiple windows and multiple facebook logins to check for users being added to the global activeUsers object, and then being removed from the object upon leaving the site
#### Any background context you want to provide?

N/A
#### What are the relevant tickets on Pivotal Tracker?

N/A
#### Provide screenshots (if appropriate).

N/A
